docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+volumes:
+  mailbox:
+services:
+  postfix:
+    build: 10/
+    image: camptocamp/odoo-postfix
+    volumes:
+      - mailbox:/var/spool/postfix
+    ports:
+      - "2525:25"
+    environment:
+      POSTFIX_HOSTNAME: odoo.postfix.local
+      ODOO_DB_NAME: odoodb
+      ODOO_HOST: odoo
+      ODOO_USER_ID: 1
+      ODOO_USER_PWD: admin
+      ODOO_MODEL: project.project
```

Comment:
by reading the docker-entrypoint.sh, it seems to me that ODOO_DB_NAME, ODOO_HOST, ODOO_USER_ID, ODOO_USER_PWD and ODOO_MODEL are not used, they are overriden by the content of `CLIENT_INSTANCE`, here:

```bash
 for client in $(echo "$CLIENT_INSTANCE" | jq -c '.[]'); do
     NAME=$(echo ${client} | jq -r '.name')
     DOMAIN=$(echo ${client} | jq -r '.domain')
     ODOO_HOST=$(echo ${client} | jq -r '.odoo_host')
     ODOO_DB=$(echo ${client} | jq -r '.odoo_db')
     ODOO_USER_ID=$(echo ${client} | jq -r '.odoo_user_id')
     ODOO_USER_PWD=$(echo ${client} | jq -r '.odoo_user_pwd')
     ODOO_MODEL=$(echo ${client} | jq -r '.odoo_model')
 
     echo "@$DOMAIN $NAME-MailGate" >> /etc/postfix/virtual_aliases
 
     CALL_MAILGATE="|/openerp_mailgate.py --host=$ODOO_HOST --dbname=$ODOO_DB -u $ODOO_USER_ID -p $ODOO_USER_PWD" # -o $ODOO_MODEL"
     echo $NAME'-MailGate: "'$CALL_MAILGATE'"' >> /etc/aliases
 
     MYDESTINATION="$MYDESTINATION, $DOMAIN"
     adduser $NAME'-MailGate' --disabled-login --system --force-badname --no-create-home
done
```

Reply:
You're right, I think we can keep `ODOO_*` var in Dockerfile (because CLIENT_INSTANCE is not definied in Dockerfile) and remove them from `docker-compose.yml` as it will be overide by CLIENT_INSTANCE var. @guewen are you agree ?

Reply:
IMO we should remove them from the Dockerfile as well because that's confusing to have them here as the values set there are never used

Reply:
> the values set there are never used

You're right, so removing with 3be2de5